### ISSUE TITLE
Deleting a word with CTRL-W copies it to clipboard

### ISSUE THREAD

**Issue by destroycomputers** [warpdotdev/Warp#544]
### Describe the bug

```shell
Whenever I press ^W to delete previous word it is getting copied into clipboard. "copy on select" is disabled.
```

### To Reproduce

Steps to reproduce:
1. Enter arbitrary text
2. Enter ^W (Ctrl+W)

### Expected behaviour

```shell
The word is deleted without clobbering the clipboard.
```

### Operating System

MacOS

### OS Version

tested on 12.0, 12.1

**Comment by elviskahoro** [comment]
This has been triaged but no ETA on release. Will circle back here when it's a WIP.

**Comment by akwodkiewicz** [comment]
This has been resolved in `v0.2023.04.25.08.05.stable_00`

<img width="536" alt="Screenshot 2023-04-28 at 16 29 57" src="https://user-images.githubusercontent.com/22861194/235176308-ca52cda6-15eb-4d3d-b483-24de42a8fd34.png">

**Comment by dannyneira** [comment]
Hey Folks! Just wanted to give you a quick update - as @akwodkiewicz mentioned, we've fixed the issue with the latest version of Warp `v0.2023.04.25.08.05.stable_00`.

I'll go ahead and mark this task as complete, but if you notice anything else, feel free to give us a heads-up. Thanks!
